plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/AppEngineProjectDeployer.java

```diff
@@ -35,21 +40,43 @@
  */
 public class AppEngineProjectDeployer {
 
+  @VisibleForTesting
+  static final List<String> APP_ENGINE_CONFIG_FILES = Collections.unmodifiableList(Arrays.asList(
```

Comment:
My understanding (which might be wrong) is that apps and these configs are different in that these are GCP project-level or service-level configurations, which will affect all apps. (Not sure if flex has a different perspective.) I could imagine that people may want to iterate over apps with frequent deploys but may not want to touch project-wide settings.

But @akerekes was also suggesting to always deploy these files anyway, so I think I'll remove the checkbox.